src/ServicePulse.Host/app/js/app.bootstrap.js

```diff
@@ -13,6 +13,9 @@
         angular.element(document).ready(function () {
             angular.bootstrap(document, ['sc']);
         });
-    }, function (error) {
+    }, function () {
+        window.document.getElementById('cantConnectMessage').style.display = 'block';
```

Comment:
Have to use DOM functions as angular isn't loaded at this point yet (which is also why the `ng-if="::false"` thing works)